frontend/src/components/GifWidget/GifWidget.styled.ts

```diff
@@ -0,0 +1,89 @@
+import styled from "styled-components";
+
+const GifWidgetStyled = styled.div`
+    position: absolute;
+    right: 3rem;
+    bottom: 7rem;
+
+    width: 30rem;
+    height: 35rem;
+    background-color: #000000bb;
+
+    box-shadow: 5px 10px 25px #000000bb;
+    border-radius: 4px;
+    overflow-y: auto;
+
+    &::-webkit-scrollbar {
+        width: 5px;
+    }
+
+    &::-webkit-scrollbar-thumb {
+        border-radius: 3px;
+        background: ${({ theme }) => theme.primary};
+    }
+`;
+
+const GifWidgetContent = styled.div``;
+
+const GifWidgetSearch = styled.input`
+    all: unset;
```

Comment:
I think it's unnecessary, cause u don't inherit nothing, and "unset" take inherited values as default in that case anyway.

https://developer.mozilla.org/en-US/docs/Web/CSS/unset

Reply:
there is a reason why i used unset: all. please do not add comments like this one without testing the proposed solution yourself. reading the information in the link which you provided is also advised.

Reply:
I can't see where it's provided, but if it's necessary, i resolve this conversation